hooks/post_gen_project.py

```diff
@@ -134,25 +142,40 @@ def clone_gh_repo(url):
     """
     pass
 
-def load_headers(headers_path, manuscript_path):
-    """Loads usepackages.txt and newcommands.txt into manuscript.tex
-    header.
-
-    Updates manuscript.tex headers in place with the contents of the user-files.
-
+def load_headers(project_path):
+    """Find usepackages.txt ,newcommands.txt, and manuscript.tex in
+    project directory. Insert usepacakgea and new commands into
+    manuscript.tex.
 
     Parameters
     ----------
-    headers_path : Path
-      The path to the location of the usepackages.txt file
-    manuscript_path : Path
-      The path to the manuscript.tex file
+    project_path : Path
+      The path to the location of project directory.
```

Comment:
I think it would be helpful for users if you put some examples of what the user `packages.tex` file should look like and the user `newcommands.tex` file too.